ctx.go

```diff
@@ -246,6 +264,14 @@ func (c *DefaultCtx) Body() []byte {
 	return body
 }
 
+func (c *DefaultCtx) BodyJSON(v any) error {
```

Comment:
I think these methods can be a part of Bind() by chaining

Reply:
Binds already have json and XML method with extra content-type header checking

Reply:
I think you can add them into the Bind(). And you can check content type by Should/Must

Reply:
should we rename `Should/Must` to `Strict`?

Reply:
Sounds good